apps/src/levelbuilder/ai-iteration-tools/ai-tutor/AITutorTester.tsx

```diff
@@ -16,7 +16,7 @@ import {
   availableEndpoints,
```

Comment:
- [ ] get AI Tutor Tester back into a functional state (likely remove all the Gen AI chat options 🤔 )